src/vita-public.h

```diff
@@ -0,0 +1,7 @@
+#ifndef VITA_PUBLIC
+#if (defined(_WIN32) || defined(__CYGWIN__)) && defined(VITA_SHARED)
+#define VITA_PUBLIC __declspec(dllimport)
+#else
+#define VITA_PUBLIC
+#endif
+#endif
```

Comment:
It's not clear that `src/vita-public.h` means that it's for importing whereas `src/build/vita-public.h` is for exporting. I'm not sure why the folder `build` was created by the way.